src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -19,7 +19,8 @@
      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
    <% } else{ %>
      <a href="/login">Login</a>
-   <% } %>
+   <% } %>   
+   <a href="/about.jsp">About</a>
```

Comment:
Why did you add this change? In general we want to separate bug fixes and feature implementation so we can easily rollback.

Reply:
Removed.